### ISSUE TITLE
Not compatible with https://github.com/vsivsi/meteor-job-collection

### ISSUE THREAD

**Issue by prosf** [cult-of-coders/redis-oplog#246]
The meteor-job-collection( https://github.com/vsivsi/meteor-job-collection ) package is not compatible with redis-oplog.

When calling most of Meteor-Job-Collection methods ( e.g Jobs.processJobs) a  `TypeError: Cannot read property 'root' of undefined` error is thrown.

It seems this package needs Mongo.Collection unaltered in order to function properly.

**Comment by prosf** [comment]
Yes I did. This error is thrown while Redis-oplog is added on top of the packages list.

**Comment by nick-gudumac** [comment]
I experience the same issue, any chance of getting this to work together?

**Comment by theodorDiaconu** [comment]
@nick-gudumac @prosf pls create a quick reproduction repo for me and I'll take care of it. Cheers.

**Comment by nick-gudumac** [comment]
@theodorDiaconu here is the reproduction repo: https://github.com/nick-gudumac/redis-oplog-issue-246

Thank you so much!

**Comment by nick-gudumac** [comment]
Hi @theodorDiaconu! Just a friendly reminder, if you had any chance to look into this issue, perhaps there is a workaround? We rely heavily `meteor-job-collection` package, but we'd want to integrate Redis Oplog to our stack as soon as possible. Please let us know how can we help you with this, or where should we look for a fix to work on. Thanks 🙌

**Comment by SimonSimCity** [comment]
@nick-gudumac This should be fixed in version 2.0.0. Please also try switching to my fork https://github.com/simonsimcity/meteor-job-collection which adds support for Meteor <1.6.1
